docs/en/writing-running-appium/ios/ios-tvos.md

```diff
@@ -3,32 +3,42 @@
 - Appium 1.13.0+ has tvOS support via XCUITest driver
     - It does not work on Apple TV 4K since [ios-deploy](https://github.com/ios-control/ios-deploy) does not work on wireless devices
 
+<img src="https://user-images.githubusercontent.com/5511591/55161297-876e0200-51a8-11e9-8313-8d9f15a0db9d.gif" width=50%>
+
 ## Setup
 
 You can run tests for tvOS by changing the `platformName` capability like it is done below.
 
 ```json
 {
     "automationName": "XCUITest",
-    "platformName": "tvOS",
+    "platformName": "tvOS", // here
     "platformVersion": "12.2",
     "deviceName": "Apple TV",
     ...
 }
 ```
 
+Tips to build WDA for tvOS
+- Update to the latest Carthage
+- Be sure tvOS simulator exists in your simulator list if you would like to run your test on tvOS simulators
+    - e.g. `xcrun simctl list | grep "com.apple.CoreSimulator.SimRuntime.tvOS"` shows results
+    - Newer Carthage raises an error like `Could not find any available simulators for tvOS` if tvOS simulators do not exist in tvOS build
+
 ## Limitations
 Gesture commands do not work for tvOS. Some commands such as pasteboard do not work as well.
 
 We can handle `focus` on tvOS by simply pressing keys such as up/down/left/right/home.
-tvOS does actions on the _focused_ element. You can get the value of the `focus` attribute via [Attributes API](http://appium.io/docs/en/commands/element/attributes/attribute/).
+tvOS does actions on the _focused_ element. You can get the value of the `focus` attribute via [Attributes API](http://appium.io/docs/en/commands/element/attributes/attribute/). _Get active element_ API returns the focused element.
 
 
 ## Basic Actions
 
+_pressButton_ and get focused element by _get active element_ are basic actions in tvOS.
```

Comment:
pressButton and getting the focused element by get active element are the basic actions for tvOS